compiler/tools/fpp-from-xml/test/top/fprime_ref_packets.fpp

```diff
@@ -0,0 +1,5 @@
+topology T {
+
+  include "fprime_ref_packets.txt"
```

Comment:
I don't understand this change.  It includes a `.txt`?  I don't see that file anywhere in this change set.

Reply:
Good catch! The file name was wrong. There was another mistake that masked this mistake. I fixed both.